index.js

```diff
@@ -5,14 +5,42 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
-
+header.style.color = "purple"
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+let profile = document.querySelector('#profile')
+console.log(profile)
+let name = profile.querySelector('h2')
+let nickname = profile.querySelector('em')
+let image = profile.querySelector('img')
+console.log(name, nickname, image)
+name.innerText = traveler.name
+nickname.innerText = traveler.nickname
+image.src = traveler.photo
+image.alt = traveler.name
 
```

Comment:
Nice work scoping your search to the `profile`! Scoping our searches is also better performance wise because there's less DOM traversal needed to be done!